sql/ddl_internal.sql

```diff
@@ -565,3 +565,11 @@ BEGIN
     END LOOP;
 END
 $BODY$;
+
+
+--documentation of these function located in chunk_index.h
+CREATE OR REPLACE FUNCTION _timescaledb_internal.chunk_index_recreate_create(chunk_index_oid OID) RETURNS OID
+AS '$libdir/timescaledb', 'chunk_index_recreate_create' LANGUAGE C IMMUTABLE STRICT;
```

Comment:
These functions should be `VOLATILE` and not `IMMUTABLE` since they modify that database.

Reply:
good catch. done.